hoot-core/hoot-core.pro

```diff
@@ -38,8 +38,11 @@ OTHER_FILES = \
     $$files(../conf/schema/*.json, true) \
     $$files(../docs/*.asciidoc, true) \
     ../conf/core/ConfigOptions.asciidoc \
-    ../conf/core/*Conflation.conf \
-    ../conf/core/*Algorithm.conf \
+    ../conf/core/Attribute.conf \
+    ../conf/core/Diff.conf \
+    ../conf/core/Horizontal.conf \
+    ../conf/core/Network.conf \
+    ../conf/core/Reference.conf \
```

Comment:
The 5 files above shouldn't exist at all.  Using a wildcard allows us to add and update conflation and algorithm config files without having to change this project file.  These changes can be reverted.